src/coverup/coverup.py

```diff
@@ -1,20 +1,29 @@
 import asyncio
-import openai
 import json
+import litellm # type: ignore
+import logging
+import openai
 import subprocess
-from pathlib import Path
-import typing as T
 import re
 import sys
+import typing as T
+
+from pathlib import Path
 from datetime import datetime
+
 from .llm import *
 from .segment import *
 from .testrunner import *
 
 
 PREFIX = 'coverup'
-DEFAULT_MODEL='gpt-4-1106-preview'
+DEFAULT_MODEL=''    # Model logic now in main()
```

Comment:
You could just delete this now, and adjust `--model` so it it has no default.

Reply:
Done.